easybuild/main.py

```diff
@@ -163,9 +163,6 @@ def main(testing_data=(None, None, None)):
         new_umask = int(options.umask, 8)
         old_umask = os.umask(new_umask)
 
-    # set temporary directory to use
-    eb_tmpdir = set_tmpdir(options.tmpdir)
```

Comment:
I need a temporary directory to symlink the included easyblocks into the correct namespace.

When `--include-easyblocks` is combined with `--list-easyblocks`, this point is never reached (`--list-easyblocks` is handled via `tools/options.py`, and so we don't return back to `main.py` after parsing options).

I now set the temporary directory as a part of postprocessing the parsed options, so earlier than I was here. Should be fine (can't hurt to do it earlier).